apps/src/templates/teacherDashboard/EditSectionForm.jsx

```diff
@@ -1,22 +1,122 @@
+              onChange={val => this.props.handleName(val.target.value)}
+            />
+          </div>
+          <Heading3>
+            Grade
+          </Heading3>
+          <div>
+            <select
+              value = {this.props.grade}
+              onChange={val => this.props.handleGrade(val.target.value)}
+            >
+              {[""].concat(this.props.validGrades).map((grade, index) => (
+                <option key={index} value={grade}>{grade}</option>
+              ))}
+            </select>
+          </div>
+          <Heading3>
+            Course
+          </Heading3>
+          <div>
+            <p>Don't know which course to teach? Find a course from the courses page to assign a course to your section
+              later. </p>
+            <AssignmentSelector
+              ref={element => this.assignment = element}
+              primaryAssignmentIds={this.props.primaryAssignmentIds}
+              assignments={this.props.validAssignments}
+            />
+          </div>
+          <Heading3>
+            Enable Lesson Extras
+          </Heading3>
+          <div>
+            <p>When Lesson Extras is on, students will end each lesson with some bonus challenges and creative projects rather
+              than being automatically advanced to the next lesson. This feature gives students the opportunity to expand
+              their knowledge and further practice, without getting ahead of their classmates. Learn more about Lesson Extras.</p>
+            <select
+              value = {this.props.extras}
+              onChange={val => this.props.handleExtras(val.target.value)}
+            >
+              <option value="yes">Yes</option>
+              <option value="no">No</option>
+            </select>
+          </div>
+          <Heading3>
+            Enable Pair Programming
+          </Heading3>
+          <div>
+            <p>When pair programming is turned on, students can choose to work with a classmate at the same computer. Turn
+              this setting on if you want students to be able to work together while sharing progress. Learn more about
+              pair programming.</p>
+            <select
+              value = {this.props.pairing}
+              onChange={val => this.props.handlePairing(val.target.value)}
+            >
+              <option value="yes">Yes</option>
+              <option value="no">No</option>
+            </select>
+          </div>
+        </div>
+        <div>
+          <ProgressButton onClick={this.props.handleBack} text="Go Back"/>
+          <ProgressButton onClick={() => {this.props.handleSave();}} text="Save"/>
+        </div>
       </div>
     );
   }
 }
 
+export default connect(state => ({
+  validGrades: state.teacherSections.validGrades,
+  validAssignments: state.teacherSections.validAssignments,
+  primaryAssignmentIds: state.teacherSections.primaryAssignmentIds,
+  sections: state.teacherSections.sections,
+}), {})(EditSectionForm);
```

Comment:
You can completely omit the second argument (`mapDispatchToProps`) when you don't need it.